stdpopsim/cli.py

```diff
@@ -298,6 +340,11 @@ def add_simulate_species_parser(parser, species):
                 "given as an argument, show help for this map. Otherwise show "
                 "help for all available genetic maps"))
 
+    species_parser.add_argument(
```

Comment:
Probably makes more sense to put ``--help-species`` first, before ``--help-models`` ?

Or, we could structure it slightly and have this as a top-level argument, so that we type
```
$ python3 -m stdpopsim  --help-species homsap
```
rather than
```
$ python3 -m stdpopsim homsap  --help-species 
```

This would be analogous to how we're doing it with the GeneticMaps. It's easy to do, you just add it to the `top_parser` in the ``stdpopsim_cli_parser`` function.

Not a strong opinion though, just a suggestion.

Reply:
I would prefer to keep it as 
"""
python3 -m stdpopsim homsap  --help-species 
"""
because I think it better follows what the logical process of the user is. II my mind, they aren't selecting the species based on the defaults, they're selecting the species, then asking what the defaults are. They aren't going to switch to simulating with one of the other species if they really are working with human data.

Reply:
But I will swap the order of `--help-species` and `-help-models`

Reply:
SGTM!